frontend/test/pytest/test_mid_circuit_measurement.py

```diff
@@ -233,38 +282,38 @@ def circuit(x):
         assert result.shape == (shots,)
         assert jnp.allclose(result, expected)
 
-    @pytest.mark.parametrize("shots", [11000])
+    @pytest.mark.parametrize("shots", [10000])
     @pytest.mark.parametrize("postselect", [None, 0, 1])
-    @pytest.mark.parametrize("reset", [False, True])
     @pytest.mark.parametrize("measure_f", [qml.counts, qml.expval, qml.probs, qml.sample, qml.var])
     @pytest.mark.parametrize(
         "meas_obj", [qml.PauliZ(0), qml.Hadamard(0) @ qml.PauliZ(1), [0], [0, 1], "mcm"]
     )
     # pylint: disable=too-many-arguments
-    def test_dynamic_one_shot_several_mcms(
-        self, backend, shots, postselect, reset, measure_f, meas_obj
-    ):
+    def test_dynamic_one_shot_several_mcms(self, backend, shots, postselect, measure_f, meas_obj):
         """Tests that Catalyst yields the same results as PennyLane's DefaultQubit for a simple
         circuit with a mid-circuit measurement."""
         if measure_f in (qml.counts, qml.probs, qml.sample) and (
             not isinstance(meas_obj, list) and not meas_obj == "mcm"
         ):
             pytest.skip("Can't use observables with counts, probs or sample")
 
+        if measure_f == qml.var:
+            pytest.xfail(
+                "`qml.var` requires an auxiliary measurement `qml.sample(obs)`"
+                " which isn't possible in Catalyst"
```

Comment:
If this is the case shouldn't we validate that the tape doesn't have any `qml.var(obs)` present?

Reply:
Yes, will check for it.